package/lang/Default.ini

```diff
@@ -56,7 +56,7 @@ en_mail=sfaulkner@paciellogroup.com
 en_website=https://www.paciellogroup.com/resources/contrastanalyser/
 WATC=http://www.wat-c.org/
 abouttext=The Colour Contrast Analyser was developed by Jun in collaboration with Steve Faulkner.
-versiontext=Colour Contrast Analyser version 2.4
+versiontext=Colour Contrast Analyser version:
```

Comment:
In the latest version, the version number is acquired from the binary file.
![ccass](https://cloud.githubusercontent.com/assets/5038132/23248450/c318e768-f9e2-11e6-8504-70dc991dc3f5.JPG)